doc/releases/v02.00.adoc

```diff
@@ -1,25 +1,25 @@
 
-Implementation:
-----------------
-OSI primarily contains an object based environment description using the message format of the protocol buffers library developed and maintained by Google.
+== Implementation
 
-Detailed description:
----------------------
+OSI primarily contains an object based environment description using the message format of the protocol buffers library developed and maintained by Google.
 
-**Top level interfaces**:
-OSI consists of two individual top level interfaces for object data: The ``OSI::GroundTruth`` and the ``OSI::SensorData`` interface.
+== Detailed description
 
 
-- The ``OSI::GroundTruth`` interface provides an exact view on the simulated objects in a global coordinate system. This message is populated using the internally available data of the used (driving)-simulation framework and is afterwards published to external subscribers with an exporter plugin.
-- The ``OSI::SensorData`` interface describes the objects in the reference frame of a sensor for environmental perception. It is generated from a ``OSI::GroundTruth`` message and can either be used to directly connect to an automated driving function using ideal simulated data, or may serve as input for a sensor model simulating limited perception as a replication of real world sensor behavior.
+Top level interfaces::
+OSI consists of two individual top level interfaces for object data: The ``OSI::GroundTruth`` and the ``OSI::SensorData`` interface.
++
+* The ``OSI::GroundTruth`` interface provides an exact view on the simulated objects in a global coordinate system. This message is populated using the internally available data of the used (driving)-simulation framework and is afterwards published to external subscribers with an exporter plugin.
+* The ``OSI::SensorData`` interface describes the objects in the reference frame of a sensor for environmental perception. It is generated from a ``OSI::GroundTruth`` message and can either be used to directly connect to an automated driving function using ideal simulated data, or may serve as input for a sensor model simulating limited perception as a replication of real world sensor behavior.
 
-**Low level interfaces**:
+Low level interfaces::
++
 Additionally to the two top level interfaces, OSI contains an ``OSI::LowLevelData`` message. This message describes the output of a (physical) model of the measurement process before tracking and object hypothesis. It does not deal with object data, but may comprise the geometrical influence of 3D simulations e.g. generating Lidar point clouds with a ray-tracer plugin. In addition this message supports generic radar reflection lists and will be extended to supporting low level camera data in the future.
```

Comment:
```suggestion
Additionally to the two top level interfaces, OSI contains an ``OSI::LowLevelData`` message. 
This message describes the output of a (physical) model of the measurement process before tracking and object hypothesis. 
It does not deal with object data, but may comprise the geometrical influence of 3D simulations e.g. generating Lidar point clouds with a ray-tracer plugin. 
In addition, this message supports generic radar reflection lists and will be extended to supporting low level camera data in the future.
```

Reply:
added missing comma here as well as "one sentence per line"

Reply:
Do we want to change already published release notes?